data/human/sales.txt

```diff
@@ -258,7 +258,8 @@ outfitter "Common Outfits"
 	"Pilot's License"
 	"Laser Rifle"
 	"Fuel Bar Fixed Display"
-
+	"Astronomical Scanner"
```

Comment:
Maybe you'd want to move this to a Deep Sky outfitter; it makes sense for this outfit to be an "improved ramscoop", and that would mean it should be sold alongside the Ramscoop, not separately.

Reply:
Thanks, I put it there as a draft for easier testing purposes and figured someone like yourself would have a better recommendation for where it should go permanently if implemented. I have plans to add more to that particular scanner as I get better at coding.